docs/proposals/Tag-Digest-CoExist.md

```diff
@@ -0,0 +1,88 @@
+# Ratify Mutation: mutual existence of tags and digest.
+
+## Problems
+
+Current User scenarios:
+1. I want to see which version of my image is deployed but I can only see the digest in the pod image.
+   1. This results in the engineer’s time being wasted on manually mapping between the image digest and the version from the image repository.
+   1. All my observability dashboards rely on tags but now I need to manually know which tags belong to which digests and somehow also make my dashboards map between those. This is a big hassle.
+
+Other user scenarios that encompass tag mutations NOT related to the current issue which users might want:
+1. In case of the pod spec already having both tag and digest, I want to be able to mutate the tag regardless and update the digest accordingly instead of leaving the digest in place.
```

Comment:
My bad, this was a brain dump on a previous version which unfortunately made its way here. I've removed this section altogether. And yes, as you've explained in cases of both `tag@digest` from upstream, it will be left untouched by the mutator and use the existing digest.